build.sbt

```
@@ -308,6 +308,21 @@ val mtagsSettings = List(
   buildInfoKeys := Seq[BuildInfoKey](
     "scalaCompilerVersion" -> scalaVersion.value
   ),
+  Compile / unmanagedSourceDirectories := {
+    val current = (Compile / unmanagedSourceDirectories).value
+    val base = (Compile / sourceDirectory).value
+    val regex = "(\\d+)\\.(\\d+)\\.(\\d+).*".r
+    // Starting from scala 2.13.9 we need to specify
+    // Compat.scala per each patch version
+    // For this case filter out `scala-2.13` directory that comes by default
+    scalaVersion.value match {
```

```suggestion
    if (scalaVersion.value == 2.13.9) {
```
so that we only do it for Scala 2.13.9 and it's fixed later on?